citar-citeproc.el

```diff
@@ -0,0 +1,106 @@
+;;; citar-citeproc.el --- Citeproc reference support for citar -*- lexical-binding: t; -*-
+;;
+;; This file is not part of GNU Emacs.
+;;
+;; This program is free software: you can redistribute it and/or modify
+;; it under the terms of the GNU General Public License as published by
+;; the Free Software Foundation, either version 3 of the License, or
+;; (at your option) any later version.
+
+;; This program is distributed in the hope that it will be useful,
+;; but WITHOUT ANY WARRANTY; without even the implied warranty of
+;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+;; GNU General Public License for more details.
+
+;; You should have received a copy of the GNU General Public License
+;; along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+;;; Commentary:
+
+;;  Provides functions for formatting bibliographic references according to
+;;  CSL styles, using 'citeproc-el'.
+
+;;  To use: load this file, set the required directory paths
+;;  'citar-citeproc-csl-locales-dir' and 'citar-citeproc-csl-style-dirs', set
+;;  'citar-format-reference-function' to 'citar-citeproc-format-reference',
+;;  and call one of the general reference functions, either
+;;  'citar-insert-reference' or 'citar-copy-reference'.
+
+;;  To set a CSL style, either set 'cite-citeproc-csl-style' manually to the
+;;  path to the desired CSL style file or call
+;;  'citar-citeproc-select-csl-style' to choose from a style file located in
+;;  'citar-citeproc-csl-style-dirs'.
+
+;;  If a CSL style is not set before running 'citar-citeproc-format-reference',
+;;  the user will be prompted to set a style.
+
+;;  A CSL style can also be set by calling 'citar-insert-reference' or
+;;  'citar-copy-reference' with a prefix-argument.
+
+;;; Code:
+
+(require 'citar)
+(require 'citeproc)
+
+(defcustom citar-citeproc-csl-style-dirs nil
```

Comment:
Why a list?

I get that it provides more flexibility, but is that really necessary?

Consider that `org-cite-csl-styles-dir` is a string.